Parsers/ASimWebSession/Tests/Apache_HTTPServer_WebSession_SchemaTest.csv

```diff
@@ -0,0 +1,143 @@
+"(2) Info: Missing optional field [NetworkDuration]"
+"(2) Info: Missing optional field [NetworkIcmpCode]"
+"(2) Info: Missing optional field [NetworkIcmpType]"
+"(2) Info: Missing optional field [NetworkPackets]"
+"(2) Info: Missing optional field [NetworkProtocolVersion]"
+"(2) Info: Missing optional field [NetworkProtocol]"
+"(2) Info: Missing optional field [NetworkSessionId]"
+"(2) Info: Missing optional field [RuleName]"
+"(2) Info: Missing optional field [RuleNumber]"
+"(2) Info: Missing optional field [Rule]"
+"(2) Info: Missing optional field [SrcAppId]"
+"(2) Info: Missing optional field [SrcAppName]"
+"(2) Info: Missing optional field [SrcAppType]"
+"(2) Info: Missing optional field [SrcBytes]"
+"(2) Info: Missing optional field [SrcDescription]"
+"(2) Info: Missing optional field [SrcDeviceType]"
+"(2) Info: Missing optional field [SrcDvcId]"
+"(2) Info: Missing optional field [SrcFQDN]"
+"(2) Info: Missing optional field [SrcGeoCity]"
+"(2) Info: Missing optional field [SrcGeoCountry]"
+"(2) Info: Missing optional field [SrcGeoLatitude]"
+"(2) Info: Missing optional field [SrcGeoLongitude]"
+"(2) Info: Missing optional field [SrcGeoRegion]"
+"(2) Info: Missing optional field [SrcInterfaceGuid]"
+"(2) Info: Missing optional field [SrcInterfaceName]"
+"(2) Info: Missing optional field [SrcMacAddr]"
+"(2) Info: Missing optional field [SrcNatIpAddr]"
+"(2) Info: Missing optional field [SrcNatPortNumber]"
+"(2) Info: Missing optional field [SrcOriginalUserType]"
+"(2) Info: Missing optional field [SrcPackets]"
+"(2) Info: Missing optional field [SrcPortNumber]"
+"(2) Info: Missing optional field [SrcUserId]"
+"(2) Info: Missing optional field [SrcUserType]"
+"(2) Info: Missing optional field [SrcUsername]"
+"(2) Info: Missing optional field [SrcVlanId]"
+"(2) Info: Missing optional field [SrcZone]"
+"(2) Info: Missing optional field [ThreatCategory]"
+"(2) Info: Missing optional field [ThreatConfidence]"
+"(2) Info: Missing optional field [ThreatField]"
+"(2) Info: Missing optional field [ThreatFirstReportedTime]"
+"(2) Info: Missing optional field [ThreatId]"
+"(2) Info: Missing optional field [ThreatIpAddr]"
+"(2) Info: Missing optional field [ThreatIsActive]"
+"(2) Info: Missing optional field [ThreatLastReportedTime]"
+"(2) Info: Missing optional field [ThreatName]"
+"(2) Info: Missing optional field [ThreatOriginalConfidence]"
+"(2) Info: Missing optional field [ThreatOriginalRiskLevel]"
+"(2) Info: Missing optional field [ThreatRiskLevel]"
+"(2) Info: Missing optional field [UrlCategory]"
+"(2) Info: Missing optional field [UrlOriginal]"
+"(2) Info: Missing recommended alias [Hostname] aliasing non-existent column [DstHostname]"
+"(2) Info: extra unnormalized column [ClientIdentity]"
+"(2) Info: extra unnormalized column [Date]"
+"(2) Info: extra unnormalized column [EventTimeGenerated]"
+"(2) Info: extra unnormalized column [ReturnedObjectSize]"
+"(2) Info: extra unnormalized column [SrcUserName]"
```

Comment:
there are few non standard columns showing, can you please have a look?
SrcUserName would be SrcUsername
EventStartTime, EventEndTime, TimeGenerated would be mapped instead of EventTimeGenerated

Reply:
Done